MonoGame.Framework/Random.cs

```diff
@@ -0,0 +1,386 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.Xna.Framework
+{
+    public static partial class MathHelper
+    {
+        /// <summary>
+        /// Class for generating random numbers consistently when using Mono or Microsoft compilers.
+        /// </summary>
+        public class Random
```

Comment:
@tomspilman @WardBenjamin quick question.. what is the original source of this Random class? or was it black box implemented? if it was copied from somewhere we need to update the header.

As for nesting... it might be useful for it NOT to be nested but I do like the 

```new MathHelper.Random();```

as it makes it clear its not a System.Random